examples/test-site/src/components/Breadcrumbs/MenuBreadcrumbs.tsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright © 2020 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { HTMLProps } from 'react';
+import { flow } from 'lodash';
+import { withSidecarNodes, asReadOnly} from '@bodiless/core';
+import { asEditable, asBodilessLink } from '@bodiless/components'; 
+import {
+  addClasses,
+  addProps,
+  withDesign,
+  replaceWith,
+  A,
+  Span,
+  withOnlyProps,
+} from '@bodiless/fclasses';
+import {
+  asSimpleMenuBase,
+  asSimpleMenuBreadcrumbs,
+} from '@bodiless/organisms';
+
+import { asBold } from '../Elements.token'; 
+import { EditorSimple } from '../Editors';
+
+const STARTING_TRAIL_NODE_KEY = 'startingTrail';
+const FINAL_TRAIL_NODE_KEY = 'finalTrail';
+
+type SeparatorProps = HTMLProps<HTMLSpanElement> & { separator : string};
+const Separator = (props: SeparatorProps) => {
+  const { separator, ...rest } = props;
+  return <Span {...rest}>{separator}</Span>
+}
+
+const withMenuBreadcrumbsDesign = withDesign({
+  Separator: replaceWith(Separator),
+  BreadcrumbLink: flow(
+    replaceWith(
+      withSidecarNodes(
```

Comment:
>Do we need "withSidecarNodes" if we're not providing a nodeKey?

breadcrumb item design is
```
<BreadcrumbItem>
   <BreadcrumbLink>
     <BreadcrumbTitle>
   </BreadcrumbLink>
</BreadcrumbItem>
```
so without withSidecarNodes, title node path will be based on link node path and it will be incorrect

>And why aren't we providing a nodekey?

we have full tile and link nodePaths in the store
as an option we can store item node paths in the store and require providing titles and links components with node keys